WorkloadManagementSystem/PilotAgent/pilotCommands.py

```diff
@@ -359,6 +358,42 @@ def _getSecurityCFG( self ):
       self.cfg.append( "-o /DIRAC/Security/CertFile=%s/hostcert.pem" % self.pp.certsLocation )
       self.cfg.append( "-o /DIRAC/Security/KeyFile=%s/hostkey.pem" % self.pp.certsLocation )
 
+class CheckCECapabilities( CommandBase ):
+  """ Used to get  CE tags.
+  """
+  def __init__( self, pilotParams ):
+    """ c'tor
+    """
+    super( CheckCECapabilities, self ).__init__( pilotParams )
+    self.cfg = []
+
+
+  def execute( self ):
+
+    retCode, resourceDict = self.executeAndGetOutput( 'dirac-resource-get-parameters -S %s -N %s -Q %s' %
```

Comment:
Installed, configured, but at no point the parseCommandLine() is done... but in the script